### ISSUE TITLE
Feature Request: Discovery wrapper

### ISSUE THREAD

**Issue by mzac** [librenms/librenms#4306]
Would it be possible to have a discovery-wrapper.py similar to the poller-wrapper.py?  I suggest this as my librenms install has 1400 devices to poll and I've noticed some discovery jobs that run via cron are just stuck.  If devices could be polled in parallel instead of in series it would be a lot more efficient.

<pre>
[root@odin librenms]# date
Thu Sep  1 09:35:31 EDT 2016
[root@odin librenms]# ps -ef | grep -i discovery.php | grep "php /var"
root      3602  3596  1 09:00 ?        00:00:27 php /var/www/librenms/discovery.php -h all -m vlans
root     18251 18201  0 Aug31 ?        00:01:29 php /var/www/librenms/discovery.php -h all
root     25547 25493  0 Aug31 ?        00:00:47 php /var/www/librenms/discovery.php -h all
root     38547 38535  0 00:33 ?        00:00:17 php /var/www/librenms/discovery.php -h all
root     41946 41929  0 Aug31 ?        00:00:24 php /var/www/librenms/discovery.php -h all
root     44406 44403  1 08:00 ?        00:01:13 php /var/www/librenms/discovery.php -h all -m vlans
root     45722 45719  1 06:00 ?        00:02:10 php /var/www/librenms/discovery.php -h all -m vlans
root     46835 46815  0 06:33 ?        00:00:07 php /var/www/librenms/discovery.php -h all
root     51450 51400  0 Aug31 ?        00:00:30 php /var/www/librenms/discovery.php -h all
root     59926 59924  1 07:00 ?        00:01:37 php /var/www/librenms/discovery.php -h all -m vlans
</pre>

<pre>
Version info:
Commit SHA: f278a9281cf769b679b09c4fedbb41e2ca6a5b84
DB Schema: 118
PHP: 7.0.10
MySQL: 5.5.50-MariaDB
RRDTool: 1.4.8
SNMP: NET-SNMP version: 5.7.2
</pre>

**Comment by laf** [comment]
I've done a pull request which will do what you want - if you can test it that would be great.

It won't fix your 'stuck' discovery processes though but might help you identify which device is causing it.
